packages/is-shallow-equal/test/index.js

```diff
@@ -104,6 +104,13 @@ describe( 'isShallowEqual', () => {
 
 		expect( isShallowEqual( a, b ) ).toBe( true );
 	} );
+
+	it( 'returns true on arrays that are a copy of each other', () => {
+		const a = [];
+		const b = a;
+
+		expect( isShallowEqual( a, b ) ).toBe( true );
+	});
```

Comment:
Style: Should be space between the closing `}` and `)` (there was an exception here in the standards which was recently removed).